04_numeronym/jessodri/main_test.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote("[a11y K8s abc]\n")
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		assert.Equal(t, actual, expected, "got %v but wanted %v", actual, expected)
```

Comment:
The definition of [`asset.Equal()`](https://godoc.org/github.com/stretchr/testify/assert#Equal) is for expected second and actual as the third parameter. It can be called as `assert.Equal(t, expected, actual)` unless there's a reason to want a specific output message.